### PR TITLE
Entry for gdata ParseExcel vulnerability

### LINKED ISSUE #11: Some gdata versions affected by CVE on bundled Perl script
https://www.cve.org/CVERecord?id=CVE-2023-7101

The culprit script at culprit version 0.65 was bundled in {gdata} versions 2.16.1 through 2.19.0:

https://github.com/cran/gdata/blob/21146034feeea59ee869df28e04114b3049481f8/inst/perl/Spreadsheet/ParseExcel.pm#L29

https://github.com/cran/gdata/commit/53ffd7bf5e276f98a557e92e1db3a1898b44d04c#diff-917380315605d506e50ff81fe1aa1b3bf5d8b925eff3dea34f4ea463c995074d

### PR DESCRIPTION
**Pull request by MichaelChirico**
Closes #11

### CODE REVIEW

**Comment by MichaelChirico** [comment]
cc @tylfin I'm not sure the formatting requirements. I was mainly confused about how to define `fixed`, should it be the _last broken version_ or the _first fixed version_? I assumed the latter, but then should that version be included in `versions` array as well?

File: vulns/gdata/RSEC-2023-9.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    events:
+    - introduced: "2.16.1"
+    - fixed: "3.0.0"
+  versions:
+  - "2.16.1"
+  - "2.17.0"
+  - "2.18.0"
+  - "2.18.0.1"
+  - "2.19.0"
+  - "3.0.0"
```
Comment (tylfin) [inline vulns/gdata/RSEC-2023-9.yaml:22]:
```suggestion
```

**Comment by tylfin** [comment]
@MichaelChirico Thanks for adding this, the `fixed` version should be the first version in which the vulnerability no longer appears, and that version should _not_ appear in the `versions` list (as these are the affected versions). More on this format available here: https://ossf.github.io/osv-schema/

File: vulns/gdata/RSEC-2023-9.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+id: RSEC-2023-8
```
Comment (tylfin) [inline vulns/gdata/RSEC-2023-9.yaml:1]:
```suggestion
id: RSEC-2023-9
```

File: vulns/gdata/RSEC-2023-9.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  - "2.18.0.1"
+  - "2.19.0"
+references:
+- type: WEB
+  url: https://security-tracker.debian.org/tracker/CVE-2023-7101
+- type: WEB
+  url: https://github.com/r-gregmisc/gdata/issues/14
+aliases:
+- CVE-2023-7101
+published: "2023-12-28T02:15:00.000Z"
```
Comment (tylfin) [inline vulns/gdata/RSEC-2023-9.yaml:29]:
```suggestion
published: "2023-12-28T02:15:00.000Z"
modified: "2024-01-04T02:15:00.000Z"
```

**Comment by MichaelChirico** [comment]
Thanks for wrapping this up Tyler!!

**Pull request merged**